tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -136,8 +138,12 @@ protected:
    /// \return `entryNumber` if the entry was successfully loaded, `kInvalidNTupleIndex` otherwise.
    virtual NTupleSize_t LoadEntry(NTupleSize_t entryNumber) = 0;
 
-   RNTupleProcessor(const std::vector<RNTupleOpenSpec> &ntuples)
-      : fNTuples(ntuples), fNEntriesProcessed(0), fCurrentNTupleNumber(0), fCurrentEntryNumber(0)
+   RNTupleProcessor(const std::vector<RNTupleOpenSpec> &ntuples, std::unique_ptr<RNTupleModel> model)
```

Comment:
I'd write a small comment to document that the model can be null and what happens in that case